### PR TITLE
Cleanup extension and update/replace dependencies

### LINKED ISSUE #978: Alter configuration to remove Material-ui (deprecated version) and use tailwindCss, mottled-library, postcss instead (recreate existing design)

### PR DESCRIPTION
**Pull request by wirednkod**
The purpose of the PR is to prepare the UI code of the extension for the UI face-lift from @Goranch3 's `figma` designs.
In order to achieve that the following actions took place:
- Remove not used files/components (cleanup);
- Add TailwindCss configuration for preparing the replacement of Material-UI (in stages) and reducing, more efficiently, lines of code, removing styled-components dependency;
- Cleanup dependencies:
   a) Replace as many as possible Material-ui components with tailwindCss code or our own mottled-library; 
   b) Remove styled-components dependencies and replace styling with tailwindCss; 
   c) Update remaining components from material-UI 4 to material-UI 5 _Note: This is just a temporary update until all components are replaced either from TailwindCss or mottled-library - the reason why did not happen in this PR - was that it would lead to many more commits and extended time until this PR is finished_

This PR can be reviewer easier commit by commit, as they were pushed in an organised manner.

Following tasks once this is merged:
- Replace rest material-UI components and remove dependency
- Update UI based on Goran's figma designs

Fixes #978 

Popup page:
image

Options page:
image

### CODE REVIEW

File: projects/extension/src/background/index.ts [hunk 1]
```diff
@@ -29,7 +29,7 @@ export interface Background extends Window {
   uiInterface: {
     onChainsChanged: (listener: () => void) => () => void
     onSmoldotCrashErrorChanged: (listener: () => void) => () => void
-    disconnectTab: (tabId: number) => void
+    disconnectApp: (tabId: number) => void
```
Comment (tomaka) [inline projects/extension/src/background/index.ts:32]:
```suggestion
    disconnectApps: (tabId: number) => void
```

File: projects/extension/src/containers/Options.tsx [hunk 2]
```diff
@@ -259,82 +129,48 @@ const Options: React.FunctionComponent = () => {
...
             >
               {poolingLogs ? "Pause" : "Retrieve "} logs
-            </Button>
-            <Button
-              className={classes.copyClipboard}
-              variant="contained"
-              startIcon={<FileCopyIcon />}
+            </button>
+            <button
+              className="my-0 mx-2 border rounded-md px-2 bg-stone-200	hover:bg-stone-400"
```
Comment (raoulmillais) [inline projects/extension/src/containers/Options.tsx:158]: erroneous tab in the class list.  Also you're using the same class list above on line 152 without `my-0`.  Maybe putting the only extra class (`my-0`) at the end of the class list here will make that more obvious when you're scanning through the file.

File: projects/extension/src/containers/Options.tsx [hunk 3]
```diff
@@ -259,82 +129,48 @@ const Options: React.FunctionComponent = () => {
...
               onClick={() => navigator.clipboard.writeText(textifyLogs())}
             >
               Copy to clipboard
-            </Button>
-            <div className={classes.errCounter}>{errLogs.length} Errors</div>
-            <div className={classes.warnCounter}>
+            </button>
+            <div className="rounded-md bg-red-500	py-2.5 px-4">
```
Comment (raoulmillais) [inline projects/extension/src/containers/Options.tsx:163]: erroneous tab in the class list

File: projects/extension/src/containers/Options.tsx (hunk 2 above)
Comment (wirednkod) [reply projects/extension/src/containers/Options.tsx:158]: Its a very good idea to actually `autosort` all the classNames in order to be more efficient.

**Comment by tomaka** [comment]
There are some files in `.parcel-cache` in this PR. That doesn't seem correct?

**Review by tomaka — APPROVED** [review]
Well, good for me :shrug:

**Pull request merged**